cime_config/machines/cmake_macros/crayclang-scream_frontier-scream-gpu.cmake

```diff
@@ -0,0 +1,38 @@
+set(MPICC "mpicc")
```

Comment:
I thought the merge of build systems was going to get rid of all these crayclang-scream cmake files.